dialogue-core/src/main/java/com/palantir/dialogue/core/TracedChannel.java

```diff
@@ -45,7 +45,13 @@ private TracedChannel(EndpointChannel delegate, String operationName, ImmutableM
     }
 
     static EndpointChannel create(Config cf, EndpointChannel delegate, Endpoint endpoint) {
-        return new TracedChannel(delegate, "Dialogue: request", tracingTags(cf, endpoint));
+        String operationName =
+                "Dialogue: request " + endpoint.serviceName() + "#" + endpoint.endpointName() + meshSuffix(cf.mesh());
```

Comment:
What is the point of the `meshSuffix`?

Reply:
Makes it obvious when we're running on the service mesh, since the behavior is likely to be very different if retries are handled by an intermediate proxy.